### ISSUE TITLE
Theme broken with new Discord update

### ISSUE THREAD

**Issue by iblowmymind** [Snapperito/Settings-Icons#24]
Discord decided to rename, like, 90% of its CSS classes, so the theme doesn't work at the moment.

**Comment by Snapperito** [comment]
mechanically the project is dead, but it can be confined to a single file pretty easily. ill look into the class reroll and update this repo. but i cant promise all theme developers will update their source to match the new files. thanks for this info <3 

ill also be able to refine and speed up the source, support multiple mods, and all existing plugins.

**Comment by Snapperito** [comment]
il be able to update the project over the next few days. ill make a post w/ it in the powercord discord server, and contact all known devs that are utilizing the project with the updated import information. 

[powercord discord](https://discord.gg/pYBzPPVt)

**Comment by DracoClaw** [comment]
There is also the new [Settings-Icons](https://github.com/CreArts-Community/Settings-Icons) by CreArts, if you're interested. Because apparently we need multiples of everything. /s

**Comment by Snapperito** [comment]
> There is also the new [Settings-Icons](https://github.com/CreArts-Community/Settings-Icons) by CreArts, if you're interested. Because apparently we need multiples of everything. /s

yeah i saw that, im still up with the intention of cross-client support (with every client), and per-plugin icons, which seems to not be the direction of that project. really just updating the project so people can use it easily <3

**Comment by DracoClaw** [comment]
Awesome! Can't wait to see what you do!
